libopenage/renderer/opengl/vertexrenderer.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <memory>
+
+#include <SDL2/SDL.h>
+
+#include "context.h"
+#include "render_target.h"
+#include "vertex_array.h"
+#include "buffer.h"
+#include "indexbuffer.h"
+#include <eigen3/Eigen/Dense>
+#include "shader_program.h"
+#include "sprite_2.h"
+
+#define RENDERER_MAX_SPRITES 100000
+#define RENDERER_VERTEX_SIZE 40
+#define RENDERER_SPRITE_SIZE RENDERER_VERTEX_SIZE*4
+#define RENDERER_BUFFER_SIZE RENDERER_SPRITE_SIZE * RENDERER_MAX_SPRITES
+#define RENDERER_INDICES_SIZE RENDERER_MAX_SPRITES*6
+
+#define SHADER_VERTEX_INDEX 0
+#define SHADER_COLOR_INDEX 1
+
+
+namespace openage{
+namespace renderer{
+namespace opengl{
+
+    struct VertexData{
+       float x;
+       float y;
+       float r;
+       float g;
+       float b;
+       float a;
+       float u;
+       float v;
+    };
+        
+    class VertexRenderer{
```

Comment:
What does this class do? Again no explanation :cry:

Reply:
this is just a test class. Was trying out different types of rendering such as static draw and such.

Reply:
Ok, if it's a test class then presumably we won't need to include it in the final code, so there's no need to spend time commenting it.